QuizApp/App/Common/AppDelegate+Injection.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+import Factory
+
+extension Container {
+
+    // MARK: - Common
+
+    static let coordinator = Factory(scope: .shared) { Coordinator() as CoordinatorProtocol }
+    static let navigationController = Factory(scope: .shared) { UINavigationController() }
+
+    // MARK: - Services
+    static let networkClient = Factory { NetworkClient() as NetworkClientProtocol }
+    static let secureStorage = Factory { SecureStorage() as SecureStorageProtocol }
+    static let checkNetworkClient = Factory { CheckNetworkClient() as CheckNetworkClientProtocol }
```

Comment:
What is the scope of these and other objects that don't have explicitly defined scope?

Reply:
Added singleton scope to those objects, otherwise, they are given unique scope.